optaplanner-persistence/optaplanner-persistence-jsonb/src/test/java/org/optaplanner/persistence/jsonb/api/score/buildin/bendable/BendableScoreJsonbAdapterTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jsonb.api.score.buildin.bendable;
+
+import javax.json.bind.annotation.JsonbTypeAdapter;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.buildin.bendable.BendableScore;
+import org.optaplanner.persistence.jsonb.api.score.AbstractScoreJsonbAdapterTest;
+
+public class BendableScoreJsonbAdapterTest extends AbstractScoreJsonbAdapterTest {
+
+    @Test
+    public void serializeAndDeserialize() {
+        assertSerializeAndDeserialize(null, new TestBendableScoreWrapper(null));
+        BendableScore score = BendableScore.of(new int[]{1000, 200}, new int[]{34});
+        assertSerializeAndDeserialize(score, new TestBendableScoreWrapper(score));
+        score = BendableScore.ofUninitialized(-7, new int[]{1000, 200}, new int[]{34});
+        assertSerializeAndDeserialize(score, new TestBendableScoreWrapper(score));
+    }
+
+    public static class TestBendableScoreWrapper extends TestScoreWrapper<BendableScore> {
+
+        @JsonbTypeAdapter(BendableScoreJsonbAdapter.class)
+        private BendableScore score;
+
+        @SuppressWarnings("unused")
+        public TestBendableScoreWrapper() {
```

Comment:
Suggestion: put a comment in the empty constructor, saying something like "Required by JSON-B." People won't have to spend time thinking why the unused constructor is there.